src/wwwroot/app/services/RolesService.js

```diff
@@ -1,199 +1,58 @@
-      {
-         id: 14,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Vacancies'
-      },
-      RemoveVacancy :
-      {
-         id: 15,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Vacancies'
-      },
-      AddCandidateToVacancy :
-      {
-         id: 16,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Vacancies'
-      },
-      RemoveCandidateFromVacancy :
-      {
-         id: 17,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Vacancies'
-      },
-      AddCandidate :
-      {
-         id: 18,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      EditCandidate :
-      {
-         id: 19,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      RemoveCandidate :
-      {
-         id: 20,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      ViewListOfCandidates :
-      {
-         id: 21,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      SearchCandidatesInInternalSource :
-      {
-         id: 22,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      SearchCandidatesInExternalSource :
-      {
-         id: 23,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Candidates'
-      },
-      ViewCalendar :
-      {
-         id: 24,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Calendar'
-      },
-      AddEvent :
-      {
-         id: 25,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Calendar'
-      },
-      EditEvent :
-      {
-         id: 26,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Calendar'
-      },
-      RemoveEvent :
-      {
-         id: 27,
-         description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
-         group: 'Calendar'
+
+   removeRole(role) {
+      if (role.id) {
+         const additionalUrl = ROLES_URL + role.id;
+         return _HttpService.remove(additionalUrl, role);
+         _HttpCacheService.clearCache(ROLES_URL);
+      } else {
+         _LoggerService.debug('Can\'t remove role', role);
+         return _$q.reject();
```

Comment:
I think we need to show notification with some error message. In this case empty reject will satisfy our needs